README.md

```diff
@@ -0,0 +1,308 @@
+
+block__element-modifier
+
+Always try to make class names meaningful -- "large" is better than 600, "primary" is better than blue.
+
+```css
+//less
+
+.search__entity-list {
+   .list-reset;
+}
+
+.search__entity-list-item {
+   padding: 10px 5px 10px 38px;
+   background-image: url(/Manager/static-images/search-sprite.png);
+   background-repeat: no-repeat;
+   font-size: 12px;
+}
+
+.search__entity-list-item--active {
+   background-color: @focusHighlight;
+}
+
+//view
+
+<ul class="search__entity-list">
+  <li class="search__entity-list-item  
+    search__entity-list-item--{{item.category}}"
+    ng-repeat="item in entity.item"
+    ng-class="{ 'search__entity-list-item--active': item === search.activeItem }"
+    ng-mouseover="search.activeItem = item">
+</ul>
+```
+
+It’s better to separate out what is essential to all form columns (positioning and gutters) and what is a specific modifier (width).
+
+```css
+//Avoid - does too many things
+
+.formColumnSmall {
+    float: left;
+    margin: 0 @grid-gutter-width / 2;
+    width: 90px;
+}
+
+//Instead, seperate it out...
+
+.form__column {
+   float: left;
+   margin: 0 @grid-gutter-width / 2;
+}
+
+.form__column--small {
+   width: 90px;
+}
+
+.form__column--default {
+   width: 202.5px;
+}
+```
+
+## Positioning/Layout
+
+Seperate positioning and layout from reuseable components.  We use a grid system for positioning, making things predictable and preventing a specific component's styles from affecting others.
+
+Lay out your pages using rows and columns.  For forms (the most common type of page in our application), we use a .form__column class with appropriate modifiers for width:
+
+```html
+<div class="row">
+    <div class="form__column form__column--default" bd-form-element>
+        <label class="form__label" for="required">Required</label>
+        <input class="form-element form-element--default" name="required" id="required" required ng-model="vm.formModel.required" />
+    </div>
+</div>
+
+<div class="form__vertical-spacer form__vertical-spacer--medium"></div>
+```
+
+You'll notice we handle vertical spacing using specific spacing elements.
+
+### Individual controls
+
+Individual blocks should not have positioning information or external margin/padding because this makes components difficult to adapt to varying circumstances. 
+For example, if an input element includes a bottom margin, we are forced to create more and more overrides in cases where we want a different size gutter between elements. 
+
+Instead, we strive to write components that take up the full size of their container (100% width) and do not have external margins or padding.
+
+## Consistent order of properties
+
+By keeping a consistent order of properties within a class, it’s easier to quickly read/write an existing class. Basically, it takes away the need to search through the entire class to see if a given property is set.
```

Comment:
I think I'd remove "basically." It sounds a little off-putting to me. Maybe it's just me. ;)